helms/odahu-flow-jupyterlab/values.yaml

```diff
@@ -55,7 +55,7 @@ security:
   # Type of authorization. Currently only oauth2_proxy is supported
   # Valid values are:
   # - oauth2_proxy
-  integration: oauth2_proxy
+  integration: ""
```

Comment:
why it's empty if oauth2_proxy is the only supported option ?

Reply:
Empty is valid too. By default, oauth-proxy ingress annotations will not be generated.
But I am going to remove these options. Because they are only valid for our deployment, so, they must be placed in the infra repository.